### PR TITLE
Add eikona-media/contao-isotope-akeneo-connector

### PR DESCRIPTION
**Pull request by rabauss**

### CODE REVIEW

File: meta/eikona-media/contao-isotope-akeneo-connector/logo.svg [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          </g>
+        </g>
+      </g>
+    </g>
+  </g>
+</svg>
```
Comment (fritzmg) [inline meta/eikona-media/contao-isotope-akeneo-connector/logo.svg:112]: Please minify the SVG as per the documentation.

File: meta/eikona-media/contao-isotope-akeneo-connector/composer.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  "autoload": {
+    "psr-4": {
+      "EikonaMedia\\Contao\\AkeneoConnector\\": "src/"
+    }
+  },
+  "extra": {
+    "contao-manager-plugin": "EikonaMedia\\Contao\\AkeneoConnector\\ContaoManager\\Plugin"
+  }
+}
```
Comment (fritzmg) [inline meta/eikona-media/contao-isotope-akeneo-connector/composer.json:50]: Why did you add a `composer.json`? Or is that a new feature.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: meta/eikona-media/contao-isotope-akeneo-connector/composer.json (hunk 2 above)
Comment (rabauss) [reply meta/eikona-media/contao-isotope-akeneo-connector/composer.json:50]: It's a private package and is documented here: https://github.com/contao/package-metadata/blob/main/docs/en/README.md#public-vs-privateproprietary-packages

Comment (fritzmg) [reply meta/eikona-media/contao-isotope-akeneo-connector/composer.json:50]: Thx, wasn't aware of that 😁

**Review by fritzmg — APPROVED** [review]
